### DISCUSSION TITLE
Stuck in waitlist

### DISCUSSION THREAD

**Discussion by cosstoica** [postmanlabs/postman-flows#421] [Defect]
Hello,

I've spent the last 2-3 hours creating a flow in the desktop app. After that, I closed the app and when I opened it back, I see the `Flows is Coming soon` message. However, in the web app, I can see my flow (but calling it doesn't work). 

I've reinstalled the app, even `Clear cache & Reload`, but nothing seems to bring back the flows in the desktop app.
 
<img width="386" alt="image" src="https://user-images.githubusercontent.com/109530500/203636421-10ae9775-8458-49ed-9d92-1f332fc6f91a.png">

**Comment by saswatds** [comment]
Hi @cosstoica really sorry about this. We have identified the issue and are hot fixing it right away.

**Reply by nateb-genesys** [reply]
I just ran into the same issue when updating to 10.5.4. Is the fix still in the works?

**Reply by reindervandervoort** [reply]
I just updated to 10.5.6 still the same...
I've tried[ Postman Canary](https://www.postman.com/downloads/canary/) and it works! Although it doesn't recognize most of my existing flow...
